libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -910,45 +867,45 @@ static CK_RV prvEcKeyAttParse( CK_ATTRIBUTE_PTR pxAttribute,
 
 
 /**
- * @brief Searches the PKCS #11 module's object list for label and provides handle.
- *
- * @param[in] pcLabel            Array containing label.
- * @param[in] xLabelLength       Length of the label, in bytes.
- * @param[out] pxPalHandle       Pointer to the PAL handle to be provided.
- *                               CK_INVALID_HANDLE if no object found.
- * @param[out] pxAppHandle       Pointer to the application handle to be provided.
- *                               CK_INVALID_HANDLE if no object found.
- */
-void prvFindObjectInListByLabel( uint8_t * pcLabel,
-                                 size_t xLabelLength,
-                                 CK_OBJECT_HANDLE_PTR pxPalHandle,
-                                 CK_OBJECT_HANDLE_PTR pxAppHandle )
+  * @brief Searches the PKCS #11 module's object list for label and provides handle.
+  *
+  * @param[in] pcLabel            Array containing label.
+  * @param[in] xLabelLength       Length of the label, in bytes.
+  * @param[out] pxPalHandle       Pointer to the PAL handle to be provided.
+  *                               CK_INVALID_HANDLE if no object found.
+  * @param[out] pxAppHandle       Pointer to the application handle to be provided.
+  *                               CK_INVALID_HANDLE if no object found.
+  */
+static void prvFindObjectInListByLabel( const uint8_t * pcLabel,
+                                        size_t xLabelLength,
+                                        CK_OBJECT_HANDLE_PTR pxPalHandle,
+                                        CK_OBJECT_HANDLE_PTR pxAppHandle )
 {
-    uint8_t ucIndex;
+    uint32_t ulIndex;
 
     *pxPalHandle = CK_INVALID_HANDLE;
     *pxAppHandle = CK_INVALID_HANDLE;
 
-    for( ucIndex = 0; ucIndex < pkcs11configMAX_NUM_OBJECTS; ucIndex++ )
+    for( ulIndex = 0; ulIndex < ( uint32_t ) pkcs11configMAX_NUM_OBJECTS; ulIndex++ )
```

Comment:
Is the cast to `( uint32_t )` required for `pkcs11configMAX_NUM_OBJECTS`?  I see other comparisons to `uint32_t` in this file without it (e.g. line 920).  Delete it if it's not needed.